src/platform/site-wide/sass/consolidated-patches.scss

```diff
@@ -80,7 +73,7 @@ header.merger {
     }
   }
 
-  @media (min-width: $medium-screen) {
+  @media (min-width: $teamsite-large-screen) {
     .va-notice--banner-inner,
     .va-crisis-line-container {
       width: $teamsite-width;
```

Comment:
Instances of `$teamsite-width` have been scoped within `$teamsite-large-screen`.